audit/projects/k8s-infra-prow-build-trusted/services/container/clusters/prow-build-trusted.json

```diff
@@ -48,14 +46,6 @@
     "us-central1-f",
     "us-central1-b"
   ],
-  "loggingConfig": {
-    "componentConfig": {
-      "enableComponents": [
-        "SYSTEM_COMPONENTS",
-        "WORKLOADS"
-      ]
-    }
-  },
```

Comment:
Looks like an internal change to the GKE API.

Reply:
none here, we'll keep an eye on it...